classy_bot/math_quiz.py

```diff
@@ -0,0 +1,39 @@
+from __future__ import annotations
+
+__all__ = (
+    "random_math_quiz_from_json",
+)
+
+from typing import TypedDict
+import json
+import random
+
+from .quiz import MultiChoiceQuiz
+
+
+class MathQuiz(TypedDict):
+    problem: str
+    category: str
+    options: list[list[str]]
```

Comment:
What does this `list[list[str]]` mean or look like in practice (why the inner list?) – maybe add a comment?